examples/ts/nextjs/pages/_app.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import '../lib/bugsnag'
+import Bugsnag from '@bugsnag/js'
+
+const ErrorBoundary = Bugsnag.getPlugin('react').createErrorBoundary(React);
```

Comment:
If `React` is not passed here (which is valid API usage) the plugin blows up on the server because we are currently using `window.React` as the default parameter. For this to work properly on node we would need to guard the evaluation of that expression with a `typeof window !== "undefined"` condition.